PUBLICATION.md

```diff
@@ -0,0 +1,133 @@
+# Publication Guidelines
+
+The purpose of this guide is to ensure consistency, professionalism, and proper
+attribution in all citations and references to our published reports and
+findings. These guidelines are intended to support both internal and external
+stakeholders in referencing and utilizing our work responsibly.
+
+This guide applies to all materials published by Informal Systems, Inc.,
+including but not limited to:
+
+* Security audit reports
+* Technical whitepapers 
+* Blog posts and technical write-ups 
+* Conference presentations and materials
+* Posts on Twitter, LinkedIn, Farcaster, or other social media sites
+
+By following these guidelines, we aim to:
+
+* Establish a clear and professional citation standard for our work. 
+* Facilitate proper recognition of our contributions to the blockchain security community. 
+* Prevent misuse or misrepresentation of our findings.
+
+These guidelines are not exhaustive but provide a foundation for citing and
+publishing our materials. For specific questions or additional permissions,
+please contact your project manager.
+
+## Citation Format
+
+To ensure proper attribution and consistency when referencing materials
+published by Informal Systems, Inc., please use the following citation format:
+
+**For Security Audit Reports:**
+
+```
+[Author(s)], "[Report Title]," Informal Systems, Inc., [Month Year], available at: [URL]
```

Comment:
I wonder if we should always use Informal Security as the author for security audit reports. I never understood very well the reason for adding authors to the reports.

Reply:
I agree. I don't think other companies put names (I just did a random check with Oak reports). We also should stop doing so.